OpenEdXMobile/src/main/java/org/edx/mobile/view/WebViewDiscoverDegreesFragment.java

```diff
@@ -12,4 +12,9 @@ protected String getSearchUrl() {
     protected int getQueryHint() {
         return R.string.search_for_degrees;
     }
+
+    @Override
+    protected boolean isSearchEnabled() {
+        return false;
```

Comment:
This should always be read from the configs.

Reply:
@miankhalid As mentioned in LEARNER-6917.

**Acceptance Criteria**
Search should be disabled for Degrees tab.

Reply:
That'll be handled through configurations.

> Still need to read from config?

Yes